### PR TITLE
Pi server works

### LINKED ISSUE #145: Calibrate thermister

### PR DESCRIPTION
**Pull request by Phlosioneer**
The minimal pi server is now working, along with minimal arduino code.

closes #145

### CODE REVIEW

**Comment by Phlosioneer** [comment]
I also couldn't figure out how to do the python linting.

File: PiServer/server.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+# To install pyserial:
+# sudo apt-get install python3-pip
+# sudo pip3 install pyserial
+
+from os import listdir
+from os.path import isfile, join
+import re
+
+def connectToDB():
+	connection = pymysql.connect(host="digitalgreens.cixglou4nxxh.us-east-1.rds.amazonaws.com", user="jvoves", password="digitalgreens", db="energyhill")
```
Comment (BenMatase) [inline PiServer/server.py]: Would like to pull this information out of config.json file that is in WebServer.  There are instructions in the README on how to access the json information

Comment (BenMatase) [reply PiServer/server.py]: We can also put this off to get it deployed asap

File: PiServer/server.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+			print("sent: " + command)
+
+def findSerialPort():
+	"""Looks at the list of tty ports, and returns the first arduino tty it finds."""
+	# TODO: Return an array of all arduino tty ports, to support multiple arduinos.
+	# Wildcards from https://github.com/sudar/Arduino-Makefile/blob/bc5092f25bd330051b377c13b317a25870413a24/Arduino.mk#L1121
+	
+	# Set up the tty regex for arduinos. They connect as either ttyUSB* or ttyACM*
+	pattern = re.compile("tty(USB|ACM|\.usb(serial|modem)|\.wchusbserial).*")
+	# FIXME
```
Comment (BenMatase) [inline PiServer/server.py]: Is this problematic?

File: PiServer/server.py (hunk 1 above)
Comment (Phlosioneer) [reply PiServer/server.py]: That was my thoughts. This is still intended to be hacked-together just to _work_. Now I'll be doing the clean-up, error handling, and input-parsing stuff.

File: PiServer/server.py (hunk 2 above)
Comment (Phlosioneer) [reply PiServer/server.py]: The FIXME is referring to the `return "/dev/ttyUSB0"`. The pattern-compiling is not working the way I expect and I need to debug it.

**Pull request closed without merging**